src/test/java/com/google/sps/CommentAnalysisTest.java

```diff
@@ -57,7 +58,8 @@
 
   private static final float TEST_SCORE = 0.23f;
   private static final float TEST_MAGNITUDE = 1.5f;
-
+  private static final String TEST_MESSAGE = "Test Message";
+  private static final String TEST_ID = "000";
```

Comment:
Idk why I can't comment on line 75 - 82, so I am commenting it here:
I realized that I didn't notice last time that this if statement is not necessary. As a test author you are supposed to be responsible for passing in the correct arguments to do testing, so we don't have to check again if the frequency's size and userCommentList's size are the same. We could delete that and make constructing an object simpler.